### ISSUE TITLE
A11y_SQLAzureDataStudio_Notebook_Create new notebook_ScreenReader: The change in button is not announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9366]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#NVDA;#JAWS;#MAS1.31;#MAS4.2.12; 

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.3.1, MAS4.2.12
Screen reader: JAWS, NVDA2019.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Click File then New Notebook.
3. Launch screen reader as NVDA
4. Use tab and navigate to the buttons "Trusted cell"

**Actual**
On navigating to the button "Trusted cell" using tab press enter it changes to "Not trusted" but there is no announcement of the same.
Similar for "Collapse cell" button

**Similar issue with jaws** 

**Expected:**
The change in name of the button should be announced.

**User Impact:**
Screen reader users will not know the current value of the button.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EeFWacd8lWNOjd60t8-7taUBDq5vdpnW3JCCL3BmU009aA?e=GEzPm2

**Comment by alanrenmsft** [comment]
dupe of #9406

**Comment by Roopesh-T** [comment]
above issue is not repro, so closing the bug.
**GitHub Tag**:#Closed;
![9366_Closed](https://user-images.githubusercontent.com/86601127/218737718-d927d65e-0c48-4211-845e-5ba4acaf51cc.gif)
![image](https://user-images.githubusercontent.com/86601127/218737773-d4fc2360-885c-4230-b6d4-69bb1a253878.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
